tabby-electron/src/shells/wsl.ts

```diff
@@ -49,6 +51,14 @@ export class WSLShellProvider extends ShellProvider {
             return []
         }
 
+        // make sure that this will not use the powershell profile
+        // that may take a long time to load
+        const pwsh = new PowerShell({
```

Comment:
Move into the constructor to avoid spawning a new pwsh every time the profiles menu is opened

Reply:
Done